sklearn/multiclass.py

```diff
@@ -452,6 +463,7 @@ class OneVsOneClassifier(BaseEstimator, ClassifierMixin, MetaEstimatorMixin):
     def __init__(self, estimator, n_jobs=1):
         self.estimator = estimator
         self.n_jobs = n_jobs
+        self.pairwise_indices_ = None
```

Comment:
The constructor of a scikit-learn estimators should never set attributes with a trailing `_`, it should only store hyperparameters as attributes. Attributes with a trailing `_` should only be set by the `fit` method or by a private submethod called only at `fit` time.

At test time, methods that need access to that attribute can check its presence with the `_check_fitted_model` helper.